### ISSUE TITLE
HPC deployment

### ISSUE THREAD

**Issue by andre-merzky** [ejjordan/airflowHPC#32]
Is there documentation on how Airflow (and specifically AirflowHPC) is expected to be deployed / used on HPC resources?  Like, is there a recommended way to deploy the database, and are there example scripts to deploy and run Airflow in a job allocation?

**Comment by ejjordan** [comment]
Yes! Please take a look at the readme, which should have all the instructions necessary for HPC deployment. Please let me know if you run into any issues so that I can address them.
